apps/finance/app/src/script.js

```diff
@@ -124,14 +119,18 @@ async function createStore(settings) {
         ...state,
       }
 
-      if (eventName === INITIALIZATION_TRIGGER) {
-        nextState = await initializeState(nextState, settings)
-      } else if (addressesEqual(eventAddress, vault.address)) {
+      if (addressesEqual(eventAddress, vault.address)) {
         // Vault event
         nextState = await vaultLoadBalance(nextState, event, settings)
       } else {
         // Finance event
         switch (eventName) {
+          case events.SYNC_STATUS_SYNCING:
+            nextState.isSyncing = true
+            break
+          case events.SYNC_STATUS_SYNCED:
+            nextState.isSyncing = false
+            break
```

Comment:
Small thing, could we lift these events out of the address check and do something similar to the [Voting app](https://github.com/aragon/aragon-apps/blob/master/apps/voting/app/src/script.js#L69)?